.pre-commit-config.yaml

```diff
@@ -16,26 +16,17 @@ repos:
     hooks:
     -   id: pycln
         args: [--config=pyproject.toml]
--   repo: https://github.com/PyCQA/isort
-    rev: 5.13.2
+-   repo: https://github.com/astral-sh/ruff-pre-commit
+    rev: v0.5.5
     hooks:
-    -   id: isort
-        additional_dependencies: [tomli] # to read config from pyproject.toml
--   repo: https://github.com/psf/black
-    rev: 24.4.2
-    hooks:
-    -   id: black
--   repo: https://github.com/PyCQA/flake8
-    rev: 7.0.0
-    hooks:
-    -   id: flake8
--   repo: https://github.com/asottile/pyupgrade
-    rev: v3.15.2
-    hooks:
-    -   id: pyupgrade
-        args: [--py310-plus]
+    -   id: ruff
```

Comment:
**issue:** the ruff configuration must be provided in pyproject.toml

Reply:
(and config for isort and black should be removed from pyproject.toml)

Reply:
It's done. ✅

Reply:
✅